site/snippets/email-css-breakground.php

```diff
@@ -0,0 +1,78 @@
+/* Background colors for body and tables */
+body {
+	background-color: #141414;
+}
+table.container {
+  background: #000000;
+}
+
+.footer .wrapper {
+  background: #000000;
+}
+
+hr {
+  background-color: #434142;
```

Comment:
I'd add the color property here as well. I think that fixes the white display on specific email clients.